_posts/2021-04-07-cultural-competency-in-archives-phase-one-of-the-education-campaign.md

```diff
@@ -0,0 +1,52 @@
+
+Early in the planning process, we acknowledged that the nature of these discussions would sometimes be sensitive, challenging, and require personal reflection. To make the sessions more transparent, we supplied the agenda and our slides beforehand.  We also sent out an anonymous pre-discussion survey to provide more avenues for participation outside the sessions. The survey helped us determine how to structure the sessions and gauge reactions to the learning modules and the education campaign itself.
+
+*Pre-Discussion Survey Questions:*
+
+For many, race and racism are uncomfortable topics to discuss. What makes these topics uncomfortable for you, personally? How have these learning modules impacted your ability to engage with conversations around race and racism?
+
+- The SAA Cultural Diversity Competency Course shared a definition of culture from the Institute of Medicine that describes culture as "The accumulated store of shared values, ideas (attitudes, beliefs, values, and norms), understandings, symbols, material products, and practices of a group of people”. How would you describe the culture at the RAC?
+- Did any of these courses make you think differently about an experience at the RAC? In the reading room? Reading through or writing archival description? Interacting with donors?
+- What activities, discussion questions, and/or exercises presented within the learning modules did you find the most engaging and/or challenging?
+
+## Discussion Sessions
+
+While Amy, Darren, and I organized and facilitated this phase of the education campaign, it is important to note that we were also learning.  We took the courses, answered the pre-survey questions, and shared our thoughts as part of the sessions. Our experience with the learning modules guided the way we structured the sessions in ways that encouraged an inclusive and open dialogue.  At the start of each session, we provided [ground rules](https://extension.umn.edu/public-engagement-strategies/setting-ground-rules-productive-discussions) based on guidance from the University of Minnesota, for example: seek first to understand, not to be understood, and criticize the discussion, not the person. Prior to the session,  we asked that everyone prepare an answer to an icebreaker question: “What was your biggest takeaway from the learning modules?” We wanted to provide everyone the opportunity to express their views on the modules and ensure that all voices were heard.
+
+After the initial icebreaker, we moved into breakout rooms for a larger conversation about the Rockefeller Archive Center itself.  Because we encouraged all staff to participate, we understood that there might be discomfort expressing certain views in front of higher-ups and teammates. To encourage more free discussion, we created breakout rooms groups that varied across the hierarchy of the organization and across departments. The main discussion consisted of three broad topics.
```

Comment:
```suggestion
After the initial icebreaker, we moved into breakout rooms for a larger conversation about the Rockefeller Archive Center itself.  Because we encouraged all staff to participate, we understood that there might be discomfort expressing certain views in front of managers and teammates. To encourage more free discussion, we created breakout rooms groups that varied across the hierarchy of the organization and across departments. The main discussion consisted of three broad topics.
```